contracts/Midas/MidasProtocolAdapter.sol

```diff
@@ -0,0 +1,79 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.13;
+
+import {IERC20} from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import {SafeERC20} from "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+import {CTokenInterface} from "./interfaces/CTokenInterface.sol";
+import {ICToken, ICErc20} from "./interfaces/ICErc20.sol";
+import {IComptroller} from "./interfaces/IComptroller.sol";
+
+contract MidasProtocolAdapter {
+    using SafeERC20 for IERC20;
+
+    IComptroller public immutable comptroller;
+
+    /// Payable
+    receive() external payable virtual {}
+
+    constructor(address _comptroller) {
+        comptroller = IComptroller(_comptroller);
+    }
+
+    /**
+     * @dev Internal function to mint cTokens and transfer them to the `minter`
+     *
+     * @param cTokenAddress The cToken address to mint
+     * @param asset The underlying asset address
+     * @param amount The amount of underlying asset
+     * @param minter The recipient to transfer minted cTokens
+     */
+    function mint(
```

Comment:
should this have the prefix `_`?

Reply:
https://github.com/connext/connext-integration/pull/2/commits/28e95fdca233e72e345458568393bb84d782a391